src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -159,7 +159,7 @@ export default (
   methods: {
     async submit() {
       // Clear error notifications when route is changed
-      this.$store.commit(StoreTypes.actions.RESET_NOTIFICATIONS_STATE);
+      await this.$store.dispatch(StoreTypes.actions.RESET_NOTIFICATIONS_STATE);
```

Comment:
https://vuex.vuejs.org/api/#dispatch  states, that Vue `dispatch` returns a promise. 
And `await `-construct in general is intended to wait resolving of an ( asynchronous ) statement - and returns a resolved Promise. 

But besides of that,  what would be the more implicit way to show that?

Reply:
Yeah, seems that `async` is not required here since `dispatch` always returns a promise – even though the action itself is not returning a promise. What makes it confusing is that you can declare async actions as well, in Vuex
https://vuex.vuejs.org/guide/actions.html#composing-actions

Reply:
That's one of the essences of the  wonderful world of promises.. Promises of promises of promises ....